internal/lobby/lobby.go

```diff
@@ -48,6 +48,7 @@ func NewClientID() (ClientID, error) {
 type Player struct {
 	Conn     *websocket.Conn
 	Username string
+	Score 	 int
```

Comment:
The score for a player is gonna be saved inside "Points" field on common.Game struct located in "internal/common/types".